Login Form

K-Zama opened the pull request.

K-Zama committed:
Page of registration user

K-Zama committed:
Login form

avegao commented on css/style.css:
```diff
@@ -0,0 +1,25 @@
+    background-color: #FFDA1E;
+    color: #9D5F15;
+}
+
+body.page-login {}
+
+
```
Si no lo vas a usar no lo pongas

avegao commented on index.html:
```diff
@@ -0,0 +1,43 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Ranchers&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/style.css">
```
Quítale el punto, luego en el server va a dar problemas y con el local server ya te tira bien en local.

avegao commented on index.html:
```diff
@@ -0,0 +1,43 @@
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
```
Actualiza Bootstrap

avegao commented on index.html:
```diff
@@ -0,0 +1,43 @@
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Ranchers&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Dashboard</title>
```
Pon el title correcto de esta página

avegao commented on index.html:
```diff
@@ -0,0 +1,43 @@
+            </div>
+    </main>
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
```
Actualiza Bootstrap

avegao replied:
Y revisa también jQuery y el popper

K-Zama commented:
Done!

K-Zama committed:
body.page-login delete in css

avegao commented on index.html:
```diff
@@ -4,10 +4,10 @@
-    <link rel="stylesheet" href="./css/style.css">
-    <title>Dashboard</title>
+    <link rel="stylesheet" href="/css/style.css">
+    <title>Login</title>
```
Esto mejor si pones Ruralmondo - NOMBREPAGINA

avegao replied:
Esto no lo has cambiado aunque el título de commit diga lo contrario

avegao commented on index.html:
```diff
@@ -21,22 +21,22 @@
         <div class="row d-flex justify-content-center">
             <div class="col-12 col-md-3">
-                <form action="#">
+                <form>
```
El form de bootstap no tiene una clase? Creo que aqui te falta algo

avegao replied:
Ya he visto que no, antiguamente si hacía falta

avegao reviewed (changes requested):
Más comentario

K-Zama committed:
Title of login-page

K-Zama committed:
Title change

avegao commented on index.html:
```diff
@@ -7,7 +7,7 @@
     <link href="https://fonts.googleapis.com/css2?family=Ranchers&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="/css/style.css">
-    <title>Login</title>
+    <title>Ruralmondo-Login</title>
```
Separado!

K-Zama committed:
Space done

K-Zama committed:
Validar los datos en el login

avegao commented on js/app.js:
```diff
@@ -0,0 +1,38 @@
+        }, 2000)
+    } else {
+        spiner.style.display = "flex";
+        const bienvenida = document.createElement("h5");
```
Los nombres en inglés

avegao commented on js/app.js:
```diff
@@ -0,0 +1,38 @@
+    } else {
+        spiner.style.display = "flex";
+        const bienvenida = document.createElement("h5");
+        bienvenida.innerHTML = `¡Coge tu 9 de Airshoft con cañón turcao!`;
```
Aquí tienes una errata

avegao commented on js/app.js:
```diff
@@ -0,0 +1,38 @@
+
+const usernameLogin = document.querySelector("#username");
+const passwordLogin = document.querySelector("#password");
+const formularioLogin = document.querySelector("#formularioLogin");
```
Las variables en inglés y lo mismo con las clases y los ids de HTML o CSS, **siempre** todo en inglés. Menos los textos lógicamente.

avegao reviewed (changes requested):
Tienes comentarios

K-Zama committed:
Cambios hechos

La errata que señalas está escrita así a posta, es una tontería que les va a hacer gracia, luego te explico de donde sale.

avegao commented on js/app.js:
```diff
@@ -0,0 +1,38 @@
+    } else {
+        spiner.style.display = "flex";
+        const wellcome = document.createElement("h5");
+        wellcome.innerHTML = `¡Coge tu 9 de Airshoft con cañón turcao!`;
```
La errata no la has corregido

K-Zama replied:
Está escrito así a posta, luego te digo de donde viene. Es una tontería que se que les va a hacer gracia, por eso lo he puesto
